packages/waku/src/lib/builder/build.ts

```diff
@@ -76,17 +74,16 @@ const onwarn = (warning: RollupLog, defaultHandler: LoggingFunction) => {
   defaultHandler(warning);
 };
 
-const hash = (fname: string) =>
-  new Promise<string>((resolve) => {
-    const sha256 = createHash('sha256');
-    sha256.on('readable', () => {
-      const data = sha256.read();
-      if (data) {
-        resolve(data.toString('hex').slice(0, 9));
-      }
-    });
-    createReadStream(fname).pipe(sha256);
-  });
+const hash = async (fname: string): Promise<string> => {
+  const encoder = new TextEncoder();
+  const data = encoder.encode(fname);
```

Comment:
This seems wrong. We need to create a hash from the file content, not the file name.